src/Sql/dbo/Stored Procedures/ProviderUserProviderOrganizationDetails_ReadByOrganizationIdStatus.sql

```diff
@@ -0,0 +1,15 @@
+﻿CREATE PROCEDURE [dbo].[ProviderUserProviderOrganizationDetails_ReadByOrganizationIdStatus]
+    @OrganizationId UNIQUEIDENTIFIER,
+    @Status TINYINT
+AS
+BEGIN
+    SET NOCOUNT ON
+
+SELECT
+    *
+FROM
+    [dbo].[ProviderUserProviderOrganizationDetailsView]
```

Comment:
This view returns a whole bunch of columns we don't need, and JOINs the `Organization` table unnecessarily. We can just return the `ProviderUser` object.